conflux/src/endpoint/conflux.ts

```diff
@@ -0,0 +1,84 @@
+import { Requester, Validator } from '@chainlink/external-adapter'
+import { AdapterRequest, AdapterResponse } from '@chainlink/types'
+import { Conflux } from 'js-conflux-sdk'
+import { ethers } from 'ethers'
+import { Config } from '../config'
+
+const sendFulfillment = async (
+  provider: any,
+  from: any,
+  to: string,
+  dataPrefix: string,
+  functionSelector: string,
+  value: number,
+) => {
+  const dataPrefixBz = ethers.utils.arrayify(dataPrefix)
+  const functionSelectorBz = ethers.utils.arrayify(functionSelector)
+  const valueBz = ethers.utils.zeroPad(ethers.utils.arrayify(Number(value)), 32)
+  const data = ethers.utils.concat([functionSelectorBz, dataPrefixBz, valueBz])
+
+  const tx = {
+    from: from,
```

Comment:
Is this working? Right now `from` seems to be of the `Account` type, I guess it should be the account address. (`from.address`)

Reply:
the account also work here, but you are right, account address is better here